modules/ossm-security-cert-manage.adoc

```diff
@@ -10,7 +10,7 @@ By default, {ProductName} generates a self-signed root certificate and key and u
 .Prerequisites
 
 * Install {ProductName} with mutual TLS enabled to configure certificates.
-* This example uses the certificates from the link:https://github.com/maistra/istio/tree/maistra-2.0/samples/certs[Maistra repository]. For production, use your own certificates from your certificate authority.
+* This example uses the certificates from the link:https://github.com/maistra/istio/tree/maistra-2.1/samples/certs[Maistra repository]. For production, use your own certificates from your certificate authority.
```

Comment:
nit: Feel free to update any other references to the `maistra-2.0` branch in this file. It doesn't need to be in this PR, it can be done in a follow up.

Reply:
@jwendell thanks for catching this for me! I've updated  the path in two places. I also put this topic on the agenda for the next doc team meeting so we can discuss the issue further and determine what our next steps might be.